source/ada/lsp-ada_documents.adb

```diff
@@ -715,19 +1017,37 @@ package body LSP.Ada_Documents is
          --  diff for a part of the document
 
          Out_Span := Self.To_LSP_Range (Out_Sloc);
-         Diff
-           (Self,
-            VSS.Strings.Conversions.To_Virtual_String (S.all),
-            Span,
-            Out_Span,
-            Edit);
+
+         if Span.last.line - Span.first.line > 5 then
```

Comment:
Need a comment explaining this threshold

Reply:
You have a space at the end of comment, I guess